commands/deploy_test.go

```diff
@@ -90,6 +91,60 @@ func Test_deploy(t *testing.T) {
 		faasCmd.Execute()
 	})
 
+	if found, err := regexp.MatchString(`(?m:Deploying: test-function.)`, stdOut); err != nil || !found {
+		t.Fatalf("Output is not as expected:\n%s", stdOut)
```

Comment:
These messages are hard to work with when they appear in an error log.

Can you please include the desired regex in the message? i.e. "Tried to match regex '%s' but couldn't, got: %s"